dask/dataframe/tests/test_optimize_dataframe.py

```diff
@@ -44,3 +46,48 @@ def test_optimize_blockwise():
     graph = optimize_blockwise(ddf.dask)
 
     assert len(graph) <= 4
+
+
+def test_blockwise_optimize_toggle():
```

Comment:
I'm a bit confused as to why this test doesn't fail with dask-expr enabled?

Reply:
This doesn't go through the dask-expr optimised, you'll end up with an un-optimized expression that is materialised and then goes through the dask/dask blockwise optimise

You would have to call ddf.optimize() if you want to go through the dask-expr optimiser. We should probably fix this FWIW in here as well